templates/hotgraphicPopup.hbs

```diff
@@ -0,0 +1,42 @@
+<div class="hotgraphic-popup-toolbar component-item-color clearfix">
+    {{#unless _hidePagination}}
+    <div class="hotgraphic-popup-nav component-item-color">
+        <button class="base hotgraphic-popup-controls back" aria-label="{{_globals._accessibility._ariaLabels.previous}}" role="button">
+            <div class="hotgraphic-popup-arrow-l component-item-text-color icon icon-controls-small-left"></div>
+        </button>
+        <div class="hotgraphic-popup-count component-item-text-color">
+            <span class="current">1</span>/<span class="total">{{_items.length}}</span>
+        </div>
+        <button class="base hotgraphic-popup-controls next" aria-label="{{_globals._accessibility._ariaLabels.next}}" role="button">
+            <div class="hotgraphic-popup-arrow-r component-item-text-color icon icon-controls-small-right"></div>
+        </button>
+    </div>
+    {{/unless}}
+    <button class="base hotgraphic-popup-done" aria-label="{{_globals._accessibility._ariaLabels.closePopup}}" role="button">
```

Comment:
this close button needs to be the last element in the dom order so that the tabbing order is more relevant

Reply:
done